src/test/java/net/apnic/whowas/types/ParsingTest.java

```diff
@@ -13,11 +16,27 @@ public void testParseInterval() throws Exception {
         Parsing.parseInterval("2403:AD00:1000::/48");
         Parsing.parseInterval("::");
         Parsing.parseInterval("2403:ad00:1000::1");
+        Parsing.parseInterval("::2004");
+        Parsing.parseInterval("::3");
```

Comment:
{suggestion} Add the edge cases? eg, for v6, "::/0", "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/128", and 7fff/8000 slash 0/63/64/127/128, etc?